mne/utils/docs.py

```diff
@@ -4618,6 +4618,13 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     The title of the generated figure. If ``None`` (default), no title is
     displayed.
 """
+
+docdict["title_stc"] = """
+title : str | None
+    Title for the figure and the subject name. If None, only the subject name will be
+    used.
```

Comment:
this controls *window* title (in the GUI titlebar), not *figure* title (text on the figure image itself). So this text should be clarified, and also be more clear that the passed `title` will be *prepended* to the subject name. However, see other comment below; I'm not convinced that is the right behavior.

```suggestion
    Title for the figure; will be prepended to the subject name (separated by a comma).
    If ``None``, only the subject name will be used.
```